modules/artifact/src/main/java/org/mule/runtime/module/artifact/api/classloader/FilteringArtifactClassLoader.java

```diff
@@ -72,6 +75,18 @@ public FilteringArtifactClassLoader(ArtifactClassLoader artifactClassLoader, Cla
     this.exportedServices = exportedServices;
   }
 
+  /**
+   * Creates a new filtering classLoader
+   *
+   * @param artifactClassLoader artifact classLoader to filter. Non null
+   * @param filter filters access to classes and resources from the artifact classLoader. Non null
+   * @param exportedServices service providers that will be available from the filtered class loader. Non null.
+   */
+  public FilteringArtifactClassLoader(ArtifactClassLoader artifactClassLoader, ClassLoaderFilter filter,
+                                      List<ExportedService> exportedServices) {
+    this(getSystemClassLoader(), artifactClassLoader, filter, exportedServices);
```

Comment:
Should just be null here? Parent can be null and it assigns the correct class leader. I'm wondering if this will work on standalone

Reply:
No, the java ClassLoader doesn't seem to handle a null parent from what i see in the code. This what i am doing here would be the same as it is doing:

```
    protected ClassLoader() {
        this(checkCreateClassLoader(), getSystemClassLoader());
    }
```